vrtool_docs/Preprocessing/Genereren_database.md

```diff
@@ -1,45 +1,48 @@
-|--building_csv_path |Link naar het CSV-bestand van de gegenereerde gebouwen. Dit is verplichte invoer.|
-|--output_dir |Link naar de map waar de resultaten (database-bestand en config.json) terecht moeten komen. Dit is verplichte invoer.|
-|--output_db_name |Hier vul je een naam in voor de te creëren database. Belangrijk om “.db” aan het einde toe te voegen. Voorbeeld: --output_db_name “resultaten_trajectX.db”. Dit is verplichte invoer.|
-|--hr_input_csv |Link naar de ingevulde CSV (HR_default.csv) die gebruikt is voor de HR sommen. Dit is verplichte invoer.|
-|--waterlevel_results_path |Link naar de map met de resultaten van de waterstandssommen. Dit is verplichte invoer.|
-|--overflow_results_path	|Link naar de map met de resultaten van de overslagsommen. Dit is verplichte invoer.|
-|--piping_path	|Link naar de ingevulde CSV voor piping (Piping_default.csv). Dit is verplichte invoer, indien piping een relevant faalmechanisme is voor dit traject en de CSV is ingevuld.|
-|--stability_path	|Link naar de ingevulde CSV voor stabiliteit (Stabiliteit_default.csv). Dit is verplichte invoer, indien stabiliteit een relevant faalmechanisme is voor dit traject en de CSV is ingevuld.|
-|--revetment_path	|Link naar de uitvoermap van de bekledingssommen. Dit is alleen verplichte invoer, indien bekleding een relevant faalmechanisme is voor dit traject en er invoer voor bekleding is gegenereerd.|
 
-
-
-**Stap 5**:	Run de preprocessing voor maak_database, vul de juiste paden op de juiste plek in. Zie hier een voorbeeld waarbij de paden uit de tabel zijn gebruikt:
+### Voorbeeld invoer
+Onderstaand is een voorbeeld opgenomen met de paden uit de tabel:
 ```
 python -m preprocessing maak_database --traject_id "53-1" --vakindeling_geojson "c:\VRM\test_database\resultaten\preprocessing_vakindeling\Optie_1\Vakindeling_53-1.geojson" --characteristic_profile_csv "C:\VRM\test_database\resultaten\vakindeling1\selectie_profielen_1\selected_profiles.csv" --building_csv_path "C:\VRM\test_database\resultaten\vakindeling1\building_count_traject53-1.csv" --output_dir "C:\VRM\test_database\Output" --output_db_name "53-1_database.db" --hr_input_csv "C:\VRM\test_database\resultaten\HR_53-1.csv" --waterlevel_results_path "C:\VRM\test_database\resultaten\preprocessing_waterlevel" --overflow_results_path "c:\VRM\test_database\resultaten\preprocessing_overflow" --piping_path "c:\VRM\test_database\resultaten\Piping_53-1.csv" --stability_path "c:\VRM\test_database\resultaten\Stabiliteit_53-1.csv"
 ```
+
+### Mogelijke foutmeldingen
+
+#### Doorsnedenamen niet ingevuld in geojson
+In sommige gevallen wordt gedurende het werkproces eerst een ```geojson``` van de vakindeling gemaakt waarin de doorsnedenamen nog niet zijn opgegeven. Voor het koppelen van de informatie van mechanismen aan de dijkvakken moet deze kolommen gevuld zijn, en moet de ```geojson``` dus opnieuw worden gemaakt, met deze informatie.
```

Comment:
Inderdaad. We hebben nu (nog) geen scripts om dat automatisch te doen. Maar ik ga ervan uit dat de gebruiker de vakindeling.csv gelijk zo goed als mogelijk is invult